demo/src/app/pages/modules/select/select.page.ts

```diff
@@ -197,6 +197,11 @@ export class SelectPage {
                                  "Must return a <code>Promise</code>. " +
                                  "This must be defined as an arrow function in your class."
                 },
+                {
+                    name: "compareWith",
```

Comment:
Could you name this function `optionsCompare` please to get it inline with `optionsLookup`?